assets/src/dashboard/app/views/editorSettings/publisherLogo/gridItem.js

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { forwardRef, useMemo } from 'react';
+import { __, sprintf } from '@web-stories-wp/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+  CenterMutedText,
+  GridItemButton,
+  GridItemContainer,
+  Logo,
+} from '../components';
+import { THEME_CONSTANTS } from '../../../../../design-system';
+import PopoverLogoContextMenu from './popoverLogoContextMenu';
+
+export const GridItem = forwardRef(
```

Comment:
this code is mostly copied from `assets/src/dashboard/app/views/editorSettings/publisherLogo/index.js`. I've marked the new code.